services/movie_session.py

```diff
@@ -1,44 +1,37 @@
-from django.db.models import QuerySet
-
-from db.models import MovieSession
+import datetime
+from db.models import Movie
+from db.models import CinemaHall
+from db.models import MovieSession, Ticket
 
 
 def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
+    movie: Movie, cinema_hall: CinemaHall, show_time: datetime
 ) -> MovieSession:
-    return MovieSession.objects.create(
-        show_time=movie_show_time,
-        movie_id=movie_id,
-        cinema_hall_id=cinema_hall_id,
+    movie_session = MovieSession.objects.create(
+        movie=movie,
+        cinema_hall=cinema_hall,
+        show_time=show_time,
     )
-
-
-def get_movies_sessions(session_date: str = None) -> QuerySet:
-    queryset = MovieSession.objects.all()
-    if session_date:
-        queryset = queryset.filter(show_time__date=session_date)
-    return queryset
-
-
-def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+    return movie_session
 
 
 def update_movie_session(
-    session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
-) -> None:
+    session_id: int, **kwargs
+) -> MovieSession:
     movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
The `update_movie_session` function uses `MovieSession.objects.get()`, which will raise a `DoesNotExist` exception if the session is not found. Consider handling this exception to prevent potential crashes.